docs/other/fontsobject.rst

```diff
@@ -51,6 +51,29 @@ This example will select the first returned Font Family Array.
    alert(firstFontFamilyName+" "+firstFamilyStyle);
 
 
+----
+
+.. _FontsObject.fontsWithDefaultDesignAxes:
+
+FontsObject.fontsWithDefaultDesignAxes
+**********************************************
+
+``app.fonts.fontsWithDefaultDesignAxes``
+
+**Description**
+
+This attribute will return an array of Variable fontobject with only one per [real(?)/font dictionary(?)] where the specific Font instance will be assigned the design vector with default values defined for each axis.
```

Comment:
Oh, you really were _literal_ here in your transcribing.

> [real(?)/font dictionary(?)]

These were word suggestions - I am not sure what would be best here.